### DISCUSSION TITLE
MPD output to ALSA Loopback

### DISCUSSION THREAD

**Discussion by ElliottDyson** [MusicPlayerDaemon/MPD#1486] [Q&A]
Hi all, have my alsa loopback working perfectly, but for whatever reason, when I try to set it as an audio output for MPD to send to a DSP application, it just ignores Loopback like the device doesn't even exist. Even when using radio gui, it never appears in the selection menu of output devices to chose from. Any ideas how to sort this?
Thanks

**Comment by MaxKellermann** [comment]
https://www.musicpd.org/help/

**Comment by ElliottDyson** [comment]
> https://www.musicpd.org/help/

Thanks, but there's nothing on there related to my issue

**Reply by MaxKellermann** [reply]
It describes what is necessary to make people able to help you, but if you think enabling people to help with your issue is not related to your issue - ok for me.

**Reply by ElliottDyson** [reply]
> It describes what is necessary to make people able to help you, but if you think enabling people to help with your issue is not related to your issue - ok for me.

I wasn't being critical. That form specifically talks about bug reports and errors. This is not one of those cases. Hence, why I opened a discussion page on the GitHub.

**Reply by ElliottDyson** [reply]
By the way, if I'm wrong I'm sorry, I'm more than happy to accept that. But if you could point me to where I need to be on that page, because I clearly missed the area you were talking about.

**Reply by MaxKellermann** [reply]
> That form specifically talks about bug reports and errors. This is not one of those cases.

It says "We have a forum and an IRC channel for requesting help", and "Before you go there: [list of things to do before requesting help]" and "When you do request help, post the following data".

Where does it talk about bug reports and errors? At the very bottom, but explicitly **only** if "MPD crashes"!

I don't know if you did the list of things, but I can say for sure you did not post "the following data", because I don't see it here.

> But if you could point me to where I need to be on that page

Read and follow the whole page. If you don't understand something on this page, ask (here). If you have finished following everything on this page, come back here and post the data that was asked from you.

**Reply by ElliottDyson** [reply]
> > That form specifically talks about bug reports and errors. This is not one of those cases.
> 
> It says "We have a forum and an IRC channel for requesting help", and "Before you go there: [list of things to do before requesting help]" and "When you do request help, post the following data".
> 
> Where does it talk about bug reports and errors? At the very bottom, but explicitly **only** if "MPD crashes"!
> 
> I don't know if you did the list of things, but I can say for sure you did not post "the following data", because I don't see it here.
> 
> > But if you could point me to where I need to be on that page
> 
> Read and follow the whole page. If you don't understand something on this page, ask (here). If you have finished following everything on this page, come back here and post the data that was asked from you.

That's what I've been trying to tell you. MPD works totally fine, log file is normal etc. MPD not seeing Loopback devices as outputs is normal behaviour, I saw it elsewhere. I just want to change that, whether it's something I can change in a configuration file or something else. Log files, WILL NOT help in this situation. That's what I've been trying to say. It simply just isn't specific to my install.

**Comment by ElliottDyson** [comment]


**Reply by kingosticks** [reply]
They are probably waiting for you to provide at the very least, your mpd version and config, as stated on the help page.
